migrations/utils.js

```diff
@@ -174,8 +186,8 @@ async function migrateToNewTokenHandler(
     newHandlerInstance.address,
     tokenConfig.resourceID,
     tokenContractAddress,
-    // set decimal places if configured for token in 'local.json'
-    tokenConfig.decimals ?? emptySetResourceData
+    // set decimal places if !=18 in 'local.json'
+    tokenConfig.decimals != "18" ? tokenConfig.decimals : emptySetResourceData
```

Comment:
You added 18 to everything anyway so you can just use tokenConfig.decimals directly.

Reply:
On Solidity side if the token has 18 decimals we don't save this information, but in `local.json` we have this information because of other migrations logic. Because of thath it's a bit of a hack.